source/index.html.md

```diff
@@ -163,9 +163,9 @@ error categories, the last two digits define specific errors.
 * 0333: PIN locked. Too many attempts
 * 0334: Bitso Transfers need either an email, phone_number, or refcode specified
 * 0335: Invalid SPEI recipient name
-* 0336: No data found for CURP
-* 0337: No CURP found for data
-* 0338: Multiple CURPs found for data
+* 0336: No data found for KYC (CURP-México, CUIL-Argentina, CPF-Brasil)
+* 0337: No KYC found for data (CURP-México, CUIL-Argentina, CPF-Brasil)
+* 0338: Multiple KYCs found for data (CURP-México, CUIL-Argentina, CPF-Brasil)
```

Comment:
```suggestion
* 0336: No data found for KYC (CURP-México, CUIL-Argentina, CPF-Brazil)
* 0337: No KYC found for data (CURP-México, CUIL-Argentina, CPF-Brazil)
* 0338: Multiple KYCs found for data (CURP-México, CUIL-Argentina, CPF-Brazil)
```

Reply:
I was using Brasil based on how is it written for that country language. Buuuuuuuut I see the point that this is english documentation, in that case I should also remove é from México, just to keep it consistent. WDYT

Reply:
Oh yes  jajaja  👍🏽